packages/core-contracts/test/contracts/proxy/UUPSProxy.test.js

```diff
@@ -159,5 +159,32 @@ describe('UUPSProxy', () => {
         });
       });
     });
+
+    // TODO: Unskip tests here when the anti-bricking mechanism is put back in place.
+    describe('when trying to brick the proxy', () => {
+      describe('when trying to upgrade to a sterile implementation', () => {
+        before('deploy the implementation', async () => {
+          const factory = await ethers.getContractFactory('SterileImplementation');
+          Implementation = await factory.deploy();
+        });
+
+        it.skip('reverts', async () => {
+          await assertRevert(Instance.upgradeTo(Implementation.address), 'SterileImplementation');
+        });
+      });
+
+      describe('when trying to destroy the implementaion', () => {
+        let Destroyer;
+
+        before('deploy the malicious implementation', async () => {
+          const factory = await ethers.getContractFactory('ImplementationDestroyer');
```

Comment:
np: use the same (any) convention for naming implementations (ImplementationDestroyer vs SterileImplementation)

Reply:
Yeah, I think that the convention is to put Implementation at the end of anything that's supposed to be behind a proxy.

In this case, its not that this is not following that convention, its just not trying to. So IMO naming is good for now...